doc/source/performance/aio_page_requests.rst

```diff
@@ -4,22 +4,23 @@
 Multiple page asynchronous I/O requests
```

Comment:
Again, I don't understand the change in this section - why it needs to be changed at all?

The current text describes the multiple page readahead feature, which is an improvement in PS. The new change doesn't really explain this at all, and has some misinformation about how the (upstream) aio implementation works.

Reply:
OK.  Will fix.